src/plugins/saved_search/common/types.ts

```diff
@@ -25,9 +26,7 @@ export interface SavedSearchAttributes {
   sort: Array<[string, string]>;
   columns: string[];
   description: string;
-  grid: {
-    columns?: Record<string, DiscoverGridSettingsColumn>;
-  };
+  grid: UnifiedDataTableSettings;
```

Comment:
```suggestion
  grid: DiscoverGridSettings;
```
Rather than modifying and using the Unified Data Table interfaces, could we instead use the `DiscoverGridSettings` interface above for saved searches and the locator? Just thinking we may want to keep Unified Data Table interfaces separate from saved searches since it's a shared component.

Reply:
:+1: Makes sense, implemented in 4105d2681510c459272814d3f27e8bc7d26184ae.